### ISSUE TITLE
Support gearman with PHP 8.1

### ISSUE THREAD

**Issue by adrianbecker013** [mlocati/docker-php-extension-installer#495]
Could you please add a support gearman for PHP 8.1 version?

**Comment by mlocati** [comment]
The last gearman version doesn't support PHP 8.1: we need to wait until someone publish a new version of it.

You can track this issue here: https://github.com/php/pecl-networking-gearman/issues/12

**Comment by ro0NL** [comment]
are you saying my php8.1 upgrade is wasted due this extension? :rofl:

**Comment by ro0NL** [comment]
@mlocati commit https://github.com/php/pecl-networking-gearman/commit/7033013 works for us on 8.1, but `RUN install-php-extensions gearman-7033013` doesnt :(

would be awesome if it's supported, or we pin to commit 7033013 for the time being ...

**Comment by ro0NL** [comment]
```
31.59 ### INSTALLING REMOTE MODULE gearman ###
31.60   (installing version 7033013a1e10add4edb3056a27d62bb4708e942b)
31.62 parsePackageName(): "7033013a1e10add4edb3056a27d62bb4708e942b" is neither a valid version nor a valid state in "gearman-7033013a1e10add4edb3056a27d62bb4708e942b"
31.62 invalid package name/package file "gearman-7033013a1e10add4edb3056a27d62bb4708e942b"
```

**Comment by ro0NL** [comment]
For future referece, the working command is `RUN install-php-extensions php/pecl-networking-gearman@7033013a1e10add4edb3056a27d62bb4708e942b`

**Comment by mlocati** [comment]
Fixed by #903
